Azure-Samples/Cognitive-Services-Voice-Assistant
# Fix above lock implementation

## Purpose
Fix the implementation of above lock by stopping the app from closing on lock.

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Please test by
- interacting with the app
- lock the screen
- unlock the screen
- verify the app is still open
- lock the screen
- activate the app above lock
- click the dismiss button
- the app should not be running below lock (this is a limitation)
